pkg/ingress/controller/apisix_upstream.go

```diff
@@ -16,9 +16,8 @@ package controller
 
 import (
 	"fmt"
-	"github.com/api7/ingress-controller/pkg/ingress/apisix"
-	"github.com/api7/ingress-controller/pkg/ingress/endpoint"
-	"github.com/golang/glog"
+	"time"
```

Comment:
ditto

Reply:
It's used here
![image](https://user-images.githubusercontent.com/10428333/102193375-fba98580-3ef6-11eb-928e-5dc8be53e2e8.png)